frontend-react/src/hooks/UseLookupTable.test.ts

```diff
@@ -0,0 +1,63 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import { lookupTableServer } from "../__mocks__/LookupTableMockServer";
+import { LookupTables, ValueSet } from "../network/api/LookupTableApi";
+
+import useLookupTable, {
+    generateUseLookupTable,
+    GetLatestData,
+    GetLatestVersion,
+    getSenderAutomationData,
+} from "./UseLookupTable";
+
+jest.mock("@okta/okta-react", () => ({
```

Comment:
I don't think this mock is doing anything. I removed it and tests still pass.

Reply:
@doug-s-nava yup. resolved